PeerioTechnologies/peerio-mobile
# Feat pdf gen

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/12579/mobile-implement-new-account-key-pdf-generation 

#### Testing instructions  
go to settings - save account key

pressing the save button should open a preview of a pdf document.
